<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageRecordList</class>
 <widget class="QWidget" name="PageRecordList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="historyLayout">
    <item>
     <widget class="QPushButton" name="buttonOld2">
      <property name="text">
       <string>Analysis #2 (date)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOld1">
      <property name="text">
       <string>Analysis #1 (date)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonNew">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>New Analysis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEditProfile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Past Analyses</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
